### DISCUSSION TITLE
Preview .vercel/output with the CLI

### DISCUSSION THREAD

**Discussion by rossrobino** [vercel/community#7425] [Help]
### Summary

I've created a custom serverless function deployment that works when deploying to vercel. I'd like to preview it with the CLI locally but I'm not sure how to set it up. 

## Configuration:
Package manager: npm

Framework: Other
Build command: blank
Output dir: .vercel/output
Install: blank
Dev: blank

## Commands
- vc pull is working 
- vc build is working
- vc dev produces the following output:

```bash
Vercel CLI 34.3.1
> Removing .vercel/output
> Creating initial build
sh: yarn: command not found
```

I don't see a `preview` command, but I'm looking for a way to preview `./vercel/output` after my build.

**Comment by pawlean** [comment]
Hi @rossrobino!

Happy to help here! Have you tried the following:

1. Run `vercel --version` to check if the Vercel CLI is installed.
2. Run `vercel pull` to download your project configuration.
3. Run `vercel build` to execute without errors and create the build output in the specified directory.
4. Make sure all dependencies are installed correctly using either `npm` or `Yarn`.
5. `vercel dev` should start the server without any errors.
6. You should be able to access your local server at the provided URL (e.g., `http://localhost:3000`).

I'm also leaving some helpful and relevant documentation below:

   - [Vercel CLI Installation](https://vercel.com/docs/cli#introduction)
   - [Vercel CLI - Pull](https://vercel.com/docs/cli#commands/pull)
   - [Vercel CLI - Build](https://vercel.com/docs/cli#commands/build)
   - [Vercel CLI - Dev](https://vercel.com/docs/cli#commands/dev)
   - [Vercel Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions)
   - [Configuration - `vercel.json`](https://vercel.com/docs/project-configuration)

**Reply by rossrobino** [reply]
Hi @pawlean, thanks for your help. 

I think `vercel dev` is just running my `dev` script from my `package.json`. I'm looking for something that can run my final output located in `.vercel/output/` built according to the Vercel Build Output API. I'm not sure this is possible currently after looking into it. I know Astro does not offer preview builds when utilizing the Vercel adapter.

[a comment by an automated account is not shown]
